website/docs/docs/dbt-cloud-apis/sl-jdbc.md

```diff
@@ -403,16 +403,37 @@ semantic_layer.query(metrics=['food_order_amount', 'order_gross_profit'],
 
 ### Query with compile keyword
 
-Use the following example to query using a `compile` keyword:
+- Use the following example to query using a `compile` keyword:
+  ```sql
+  select * from {{
+  semantic_layer.query(metrics=['food_order_amount', 'order_gross_profit'],
+      group_by=[Dimension('metric_time').grain('month'),'customer__customer_type'],
+      compile=True)
+      }}
+  ```
 
-```bash
-select * from {{
-semantic_layer.query(metrics=['food_order_amount', 'order_gross_profit'],
-		group_by=[Dimension('metric_time').grain('month'),'customer__customer_type'],
-		compile=True)
-		}}
+- Use the following example to compile SQL with a [saved query](/docs/build/saved-queries). You can use this for frequently used queries.
+
+  ```sql
+  select * from {{ semantic_layer.query(saved_query="new_customer_orders", limit=5, compile=True}}
+  ```
+
+:::info A note on querying saved queries
+When querying [saved queries](/docs/build/saved-queries),you can use parameters such as `where`, `limit`, `order`, `compile`, and so on. However, note that `metric` or `group_by` parameters aren't available in this context.
+
+(SHOULD WE SAY WHY TEY CAN'T USE IT? BC IT'S PREDEFINED RIGHT? SHOULD WE LINK TO [THESE PARAMETERS](https://docs.getdbt.com/docs/dbt-cloud-apis/sl-jdbc#querying-the-api-for-metric-values)?)
```

Comment:
(SHOULD WE SAY WHY TEY CAN'T USE IT? BC IT'S PREDEFINED RIGHT? SHOULD WE LINK TO [THESE PARAMETERS](https://docs.getdbt.com/docs/dbt-cloud-apis/sl-jdbc#querying-the-api-for-metric-values) OR CREATE A NEW SECTION FOR THEM FOR GRAPHQL?)

Reply:
i'll apply this to all other callouts if there are any add'l changes

Reply:
Great call! Yes we can say: 

"`metrics` and `group_by` are fixed parameters for a saved query, since they are already pre-defined, and so we don't allow you to change those parameters at query time. If you would like to query additional metrics or dimensions, you can build the query using the standard format"